### ISSUE TITLE
GVM doesn't work on Cygwin

### ISSUE THREAD

**Issue by bodiam** [sdkman/sdkman-cli#107]
when I do this:

curl -s get.gvmtool.net | bash

I get:

Thanks for using

_____/\\\\\\**/\______**/\__/\\____________/\\_
 ___/\\//////////__\/\_______\/\_\/\\\________/\\\_
  **/\___________**\//\______/\__\/\\//\____/\\//\_
   _\/\____/\\\__\//\____/\_**\/\\///\\/\\/_\/\\_
    _\/\\_**\/////\_**\//\\**/\____\/\__\///\\/**_\/\\_
     _\/\_____**\/\_**_\//\\/\__**_\/\_**_\///__**_\/\_
      _\/\_______\/\_____\//\\______\/\_____________\/\_
       _\//\\\\\\/_______\//\_______\/\_____________\/\_
        **\////////////________**\///________\///______________\///__

```
                                   Will now attempt installing...
```

Looking for a previous installation of GVM...
Looking for JAVA_HOME...
Validating JAVA_HOME...
Looking for unzip...
Looking for curl...
Installing gvm scripts...
Create distribution directories...
Create candidate directories...
Download script archive...
Extract script archive...
unzip:  cannot find either /home/e-pragt/.gvm/tmp/res-0.9.4.zip or /home/e-pragt/.gvm/tmp/res-0.9.4.zip.zip.
Install scripts...
mv: cannot stat `/home/e-pragt/.gvm/tmp/stage/gvm-init.sh': No such file or directory
mv: cannot stat`/home/e-pragt/.gvm/tmp/stage/gvm-*': No such file or directory
Make init script executable...
chmod: cannot access `/home/e-pragt/.gvm/bin/gvm-init.sh': No such file or directory
Prime config file...
Attempt update of bash profiles...
Attempt update of zsh profiles...
Clean up local variables...

All done!

Please open a new terminal, or run the following in the existing one:

```
source "/home/e-pragt/.gvm/bin/gvm-init.sh"
```

Then issue the following command:

```
gvm help
```

Enjoy!!!

I then open a new terminal, and type gvm help:

e-pragt@PCD-1099 ~
$ gvm help
bash: gvm: command not found

I source the gvm-init:

e-pragt@PCD-1099 ~
$    source "/home/e-pragt/.gvm/bin/gvm-init.sh"
bash: /home/e-pragt/.gvm/bin/gvm-init.sh: No such file or directory

If I do a 'ls' of .gvm:

e-pragt@PCD-1099 ~/.gvm
$ ls -ltr
total 0
drwxr-xr-x+ 1 e-pragt Domain Users 0 Jan  8 14:04 bin
drwxr-xr-x+ 1 e-pragt Domain Users 0 Jan  8 14:04 src
drwxr-xr-x+ 1 e-pragt Domain Users 0 Jan  8 14:04 groovy
drwxr-xr-x+ 1 e-pragt Domain Users 0 Jan  8 14:04 grails
drwxr-xr-x+ 1 e-pragt Domain Users 0 Jan  8 14:04 griffon
drwxr-xr-x+ 1 e-pragt Domain Users 0 Jan  8 14:04 gradle
drwxr-xr-x+ 1 e-pragt Domain Users 0 Jan  8 14:04 vertx
drwxr-xr-x+ 1 e-pragt Domain Users 0 Jan  8 14:04 tmp
drwxr-xr-x+ 1 e-pragt Domain Users 0 Jan  8 14:04 ext

e-pragt@PCD-1099 ~/.gvm
$ cd bin/

e-pragt@PCD-1099 ~/.gvm/bin
$ ls -ltr
total 0

In short: it seems like GVM was not installed properly!
